### DISCUSSION TITLE
Can't add comments by self.

### DISCUSSION THREAD

**Discussion by chena1982** [docmost/docmost#441] [Help / Questions] [answered]
Sometimes i add a article from web, i want add some comments.
It seems can't add a comments by self, just show "No comments yet.", i can't find a place to add comments.
Thanks.

**Comment by Philipinho** [comment, marked as answer]
You have to highlight the text you wish to comment on and you should see the comment icon in the popup menu.

**Reply by RobsonMi** [reply]
This will only work if user has edit permissions, right? Any way we can allow users with view only permission to add comments? For instance when they notice a mistake and would like to comment on that?

**Reply by Philipinho** [reply]
@RobsonMi, it's not possible for view-only users to ad comments at this time. We plan to support it in the future.

**Reply by RobsonMi** [reply]
Super good enough :) Project looks great, fits almost perfectly to our workflow. Only these comments so far look like something that would be nice to allow for view-only users but we can wait.
